### PR TITLE
feat: Movement Spec Champion TeoViTeoVi 

### PR DESCRIPTION
**Pull request by diszell2008**
Hi Lava Foundation,

This is my submission for the Movement Spec Champion. I participate provider some chain in testnet . This is my spec chaimpions

```
+1% - support and running a Provider on testnet
+1% - creating a spec from scratch (inheritance is only worth 0.5% and is mandatory if possible)
+1% - benchmarking CU and timeouts
+1% - providing active support to Providers, if more than 20 providers are retained over 3 months
+2-3% - if an ecosystem creates an incentive pools based on your referral
```

Discord : diszell2008

### CODE REVIEW

File: mainnet-1/specs/movement.json [hunk 1]
```diff
@@ -0,0 +1,629 @@
...
+        "specs": [
+            {
+                "index": "MOVEMENT",
+                "name": "movement mainnet",
+                "enabled": true,
+                "reliability_threshold": 268435455,
+                "data_reliability_enabled": true,
+                "block_distance_for_finalized_data": 0,
+                "blocks_in_finalization_proof": 5,
+                "average_block_time": 200,
```
Comment (ranlavanet) [inline mainnet-1/specs/movement.json:14]: From their explorer it seems movement is slightly slower than Aptos. 
https://explorer.movementnetwork.xyz/blocks?network=testnet
seems like average block time is around 1 second.

File: mainnet-1/specs/movement.json [hunk 2]
```diff
@@ -0,0 +1,629 @@
...
+                "allowed_block_lag_for_qos_sync": 50,
```
Comment (ranlavanet) [inline mainnet-1/specs/movement.json:15]: change to 5

File: mainnet-1/specs/movement.json [hunk 3]
```diff
@@ -0,0 +1,629 @@
+{
+    "proposal": {
+        "title": "Add Specs: Movement",
+        "description": "Adding new specification support for relaying Movement data on Lava",
+        "specs": [
+            {
+                "index": "MOVEMENT",
+                "name": "movement mainnet",
+                "enabled": true,
```
Comment (ranlavanet) [inline mainnet-1/specs/movement.json:9]: please use inheritance to build specs, see example on lavanet repo
https://github.com/lavanet/lava/blob/main/cookbook/specs/movement.json

**Review by ranlavanet — CHANGES_REQUESTED** [review]

File: mainnet-1/specs/movement.json [hunk 4]
```diff
@@ -0,0 +1,629 @@
...
+                "reliability_threshold": 268435455,
+                "data_reliability_enabled": true,
+                "block_distance_for_finalized_data": 0,
+                "blocks_in_finalization_proof": 5,
+                "average_block_time": 200,
+                "allowed_block_lag_for_qos_sync": 50,
+                "shares": 1,
+                "min_stake_provider": {
+                    "denom": "ulava",
+                    "amount": "47500000000"
```
Comment (ranlavanet) [inline mainnet-1/specs/movement.json:19]: 5000000000

**Review by ranlavanet — APPROVED** [review]

File: mainnet-1/specs/movement.json (hunk 4 above)
Comment (ranlavanet) [inline mainnet-1/specs/movement.json:19]: 5000000000

**Review by ranlavanet — CHANGES_REQUESTED** [review]

File: mainnet-1/specs/movement.json (hunk 4 above)
Comment (diszell2008) [reply mainnet-1/specs/movement.json]: Changed to 5000000000

File: mainnet-1/specs/movement.json (hunk 1 above)
Comment (diszell2008) [reply mainnet-1/specs/movement.json]: Changed to 1s

File: mainnet-1/specs/movement.json (hunk 3 above)
Comment (diszell2008) [reply mainnet-1/specs/movement.json:9]: Added APT inheritance

**Review by ranlavanet — APPROVED** [review]

File: mainnet-1/specs/movement.json [hunk 5]
```diff
@@ -0,0 +1,632 @@
...
+                        "enabled": true,
+                        "collection_data": {
+                            "api_interface": "rest",
+                            "internal_path": "",
+                            "type": "GET",
+                            "add_on": ""
+                        },
+                        "apis": [
+                            {
+                                "name": "/",
```
Comment (ranlavanet) [inline mainnet-1/specs/movement.json:35]: If you added inheritance why we still have duplicate APIs. 
please take a look at the movement spec and make sure to understand how inheritance works. 
https://github.com/lavanet/lava/blob/main/cookbook/specs/movement.json

**Review by ranlavanet — CHANGES_REQUESTED** [review]

File: mainnet-1/specs/movement.json (hunk 5 above)
Comment (diszell2008) [reply mainnet-1/specs/movement.json:35]: i will fix it

**Review by ignacio-rsg — APPROVED** [review]

**Pull request merged**